tools/loadout_converter/loadout_converter.py

```diff
@@ -0,0 +1,68 @@
+# Converts all existing loadout `gear` listings in the `characters` table from their 'display_name's to their typepaths.
+# To use, run `python loadout_converter.py {address} {username} {password} {database}` in the console, with the {} arguments being your SQL database information.
+
+# Note: This must ONLY be run once, and ONLY if your loadout records are using the old system.
+
+import re
+import mysql.connector
+import argparse
+import json
+import time
+
+parser = argparse.ArgumentParser()
+parser.add_argument("address", help="SQL server address (use 127.0.0.1 for the current computer)")
+parser.add_argument("username", help="SQL login username")
+parser.add_argument("password", help="SQL login password")
+parser.add_argument("database", help="Database name (Default: paradise_gamedb)")
+
+args = parser.parse_args()
+db = mysql.connector.connect(host=args.address, user=args.username, password=args.password, database=args.database)
+cursor = db.cursor()
+
+startwatch = time.time()
+
+with open('loadout_converter.json') as json_file:
+	json = json.load(json_file)
+	# Format for both variables is (Key: `display_name`, Value: typepath)
+	standard_replacements = json["standard"]
+	special_replacements = json["special"]
+
+cursor.execute("SELECT `id`, `gear` FROM `characters` WHERE `gear` <> '';")
```

Comment:
```suggestion
cursor.execute("SELECT `id`, `gear` FROM `characters` WHERE `gear` != \"\"';")
```

Reply:
Any specific reason for the quotes rather than apostrophes, or is it a stylistic thing?

Reply:
Styling